Sourcery/CodeGenerated/Diffable.generated.swift

```diff
@@ -0,0 +1,103 @@
+// Generated using Sourcery 0.4.7 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+extension Enum {
+     override func diffAgainst(_ object: Any?) -> DiffableResult {
+        var results = DiffableResult()
+        guard let rhs = object as? Enum else {
+          results.append("Incorrect type, expected Enum, received: \(type(of: object))")
+          return results
+        }
+        if self.cases != rhs.cases { results.append("Incorrect cases, expected \(self.cases), received: \(rhs.cases)") }
+        if self.rawType != rhs.rawType { results.append("Incorrect rawType, expected \(self.rawType), received: \(rhs.rawType)") }
+
+        results.append(contentsOf: super.diffAgainst(object))
+        return results
```

Comment:
yeah I kind of needs to add `super` access to `Type` so we can do things like `{% if type.super.implements.Diffable %}`